SETUP.md

```diff
@@ -131,30 +131,50 @@ SPARK_WORKER_OPTS="-Dspark.worker.cleanup.enabled=true, -Dspark.worker.cleanup.a
 
 ## Setup guide for Azure Databricks
 
-### Requirements of Azure Databricks
+The setup for Azure Databricks can be completed by running the [create_and_configure_cluster.ipynb notebook](./notebooks/05_operationalize/create_and_configure_cluster.ipynb)
+on any jupyter instance that has the `requests` python module installed. It requires a few additional parameters to be configured; 
+please see the full details in that notebook.
+
+The following are the steps that notebook takes to prepare Azure Databricks.
+
+### Requirements for Azure Databricks
+
 * Runtime version 4.1 (Apache Spark 2.3.0, Scala 2.11)
 * Python 3
 
+**Please note:** This configuration is no longer available via the *Clusters* tab of the databricks interface, and you 
```

Comment:
I'm not sure if we should put this at all. I tested the O16N notebook with Databricks 5.0 / Spark 2.4 and it worked oob. @dciborow do you know what breaks due to spark version mismatch w/ the CosmosDB connector? 

We should figure out what / if anything breaks, and not recommend using a deprecated version via an API work around. We should also file this as feedback to the DB product team if it turns out we really do need an older version.

Reply:
Since we found the official [documentation](https://docs.microsoft.com/en-us/azure/cosmos-db/spark-connector) is different from the github site, and it states that the connector supports spark 2.3.x, we should update to 2.3.1 for now, and just deal with the next update / deprecation in April when it comes. This is still inconsistent with their github site, but we have a ticket there as well.

Reply:
This is going to end up conflicting with https://github.com/Microsoft/Recommenders/pull/457 

Will update setup.md once that PR is accepted and I can easily merge back with this one.

Reply:
Updated the SETUP file in #457 and that should be sufficient. The changes here should be incorporated into that one.